config/ember-try.js

```diff
@@ -26,6 +26,9 @@ module.exports = async function () {
       {
         name: 'ember-release',
         npm: {
+          dependencies: {
+            'ember-auto-import': '^2.0.0',
```

Comment:
Why not using more simply

```suggestion
            'ember-auto-import': '2',
```

Reply:
I prefer a more explicit syntax with prefix